_sass/variables/_colors.scss

```diff
@@ -72,6 +80,7 @@ $menu-link-color-mobile: #E5E6E7 !default;
 $menu-link-color-mobile-active: white !default;
 $menu-link-backgroundColor-mobile: #414042 !default;
 $menu-link-backgroundColor-mobile-active: #58595B !default;
+$menu-hover-border-bottom: #aaaaaa !default;
```

Comment:
Per our convention this should be something like `menu-link-borderBottomColor-hover`.